collectors/systemd-journal.plugin/README.md

```diff
@@ -378,31 +395,245 @@ nano /etc/systemd/system/systemd-journal-remote.service
+      CLIENT_DNS="${CLIENT_DNES[$i]}"
+      # Generate the client CSR
+      openssl req -newkey rsa:2048 -nodes -out $CLIENT_CN.csr -keyout $CLIENT_CN.key -subj "/CN=$CLIENT_CN/"
+      echo "subjectAltName = IP:$CLIENT_IP" > $CLIENT_CN.ext
+
+      # Sign the client certificate using the CA configuration
+      openssl ca -batch -config ca.conf -notext -in $CLIENT_CN.csr -out $CLIENT_CN.pem -extfile $CLIENT_CN.ext
+done
+```
+
+Keep in mind we have already produced the client certificates, we will make use of them when we will configure the clients.
+
+5. Copy the keys and the certificates into the `systemd-journal-remote`'s predefined places
+
+
+```sh
+sudo mkdir /etc/ssl/private # make sure that you havent created this folder, and use it already, you may dont want to change it's permissions
+sudo chmod 755 /etc/ssl/private
+sudo mkdir /etc/ssl/ca/ # make sure that you havent created this folder, and use it already, you may dont want to change it's permissions
+sudo chmod 755 /etc/ssl/ca
+sudo cp "${SERVER_CN}".key /etc/ssl/private/journal-remote.key # This is not predefined but we need to clarify the key
+sudo cp "${SERVER_CN}".pem /etc/ssl/certs/journal-remote.pem
+sudo cp ca.pem /etc/ssl/ca/trusted.pem
+```
+
+6. Adjust the permissions for the `systemd-journal-remote` to access them.
+
+```sh
+sudo chgrp systemd-journal-remote /etc/ssl/private/journal-remote.key
+sudo chgrp systemd-journal-remote /etc/ssl/certs/journal-remote.pem
+sudo chgrp systemd-journal-remote /etc/ssl/ca/trusted.pem
+
+
+sudo chmod 0640 /etc/ssl/private/journal-remote.key
+sudo chmod 755 /etc/ssl/certs/journal-remote.pem
+sudo chmod 755 /etc/ssl/ca/trusted.pem
+```
+
+6. Edit the `systemd-journal-remote.conf` to change the predefined key place and enable SSL.
+
+
+sh```
+sudo nano /etc/systemd/journal-remote.conf
+```
+
+You need to transform the corresponding section to something like this
+```
+[Remote]
+Seal=false
+SplitMode=host
+ServerKeyFile=/etc/ssl/private/journal-remote.key
+ServerCertificateFile=/etc/ssl/certs/journal-remote.pem
+TrustedCertificateFile=/etc/ssl/ca/trusted.pem
+```
 
 
 #### Configuring journal clients to push their logs to the server
 
-On the clients you want to centralize their logs, install `systemd-journal-remote`, configure `systemd-journal-upload`, enable it and start it with `systemctl`.
+In this section we will configure the clients/hosts to push their journal logs into the centralization server. You will install `systemd-journal-remote`, 
+configure `systemd-journal-upload` (with or without SSL), enable and start it.
+
+
+1. Install the `systemd-journal-remote` it run:
 
-To install it run:
 
 ```sh
 # change this according to your distro
 sudo apt-get install systemd-journal-remote
 ```
 
-Then, edit `/etc/systemd/journal-upload.conf` and set the IP address and the port of the server, like this:
+
+2. [With SSL] Copy to the client/hosts the self-signed certificates you created before (you created one per host)
```

Comment:
```suggestion
2. **With SSL**: Copy to the client/hosts the self-signed certificates you created before (you created one per host)
```

Personal preference, I don't want to mess with brackets in markdown as they are reserved for links..